modules/flowable-task-service/src/main/resources/org/flowable/task/service/db/mapping/entity/Task.xml

```diff
@@ -1123,6 +1126,9 @@
             <if test="orQueryObject.unassigned">
               or RES.ASSIGNEE_ IS NULL
             </if>
+            <if test="orQueryObject.withAssignee">
+              and RES.ASSIGNEE_ IS NOT NULL
```

Comment:
This should be an `or` and not an `and`.

Reply:
ohhh. i will fix this issue and let you know @filiphr.

https://github.com/flowable/flowable-engine/pull/2756 this PR is also created by me. can you please look at that?

Reply:
Fixed above issue.